### PR TITLE
Better onboarding blog post 

### PR DESCRIPTION
**Pull request by rebeccadee**
For https://app.asana.com/0/1199959077348208/1199959077348287/f

### CODE REVIEW

File: blogposts/2021/better-onboarding-prevent-codebase-overwhelm.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: "Better onboarding: How to prevent codebase overwhelm"
+description: "A data-driven look at how the rise in code complexity affects new developers joining a team, and how engineering managers can support new hires with better onboarding."
+author: Patricia No and Rebecca Dodd
+authorUrl: 
+publishDate: 2021-04-29T18:00+02:00
+tags: [blog, remote, design]
+slug: better-onboarding-how-to-prevent-codebase-overwhelm
+heroImage: https://sourcegraphstatic.com/better-onboarding-prevent-codebase-overwhelm.png
```
Comment (rebeccadee) [inline blogposts/2021/better-onboarding-prevent-codebase-overwhelm.md:9]:
```suggestion
heroImage: /blog/better-onboarding-prevent-codebase-overwhelm.png
```

File: blogposts/2021/better-onboarding-prevent-codebase-overwhelm.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Better onboarding: How to prevent codebase overwhelm"
+description: "A data-driven look at how the rise in code complexity affects new developers joining a team, and how engineering managers can support new hires with better onboarding."
+author: Patricia No and Rebecca Dodd
+authorUrl: 
+publishDate: 2021-04-29T18:00+02:00
+tags: [blog, remote, design]
```
Comment (rebeccadee) [inline blogposts/2021/better-onboarding-prevent-codebase-overwhelm.md]: @content-hacker Do we have a list of supported tags anywhere?

**Comment by rebeccadee** [comment]
@content-hacker I won't ask for your review on all posts but as this is my first PR from scratch I'd appreciate another set of eyes on it 😃

File: blogposts/2021/better-onboarding-prevent-codebase-overwhelm.md (hunk 2 above)
Comment (content-hacker) [reply blogposts/2021/better-onboarding-prevent-codebase-overwhelm.md]: Not that I know of. I'm not sure they even show up anywhere on the blog

**Review by content-hacker — APPROVED** [review]
@rebeccadee this looks good to me!

**Pull request merged**